### PR TITLE
boot: make sure we update assets for the system-seed-null role

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**

### CODE REVIEW

File: boot/assets_test.go [hunk 1]
```diff
@@ -761,14 +769,15 @@ func (s *assetsSuite) TestUpdateObserverUpdateMockedWithReseal(c *C) {
...
-	res, err = obs.Observe(gadget.ContentUpdate, gadget.SystemSeed, root, "managed-asset",
+	res, err = obs.Observe(gadget.ContentUpdate, seedRole, root, "managed-asset",
 		&gadget.ContentChange{After: filepath.Join(d, "foobar")})
 	c.Assert(err, IsNil)
 	c.Check(res, Equals, gadget.ChangeIgnore)
 
 	// everything is set up, trigger a reseal
 	resealCalls := 0
 	restore := boot.MockSecbootResealKeys(func(params *secboot.ResealKeysParams) error {
+		fmt.Println("reseal call")
```
Comment (mvo5) [inline boot/assets_test.go:780]: This looks like a debug leftover?

**Review by mvo5 — APPROVED** [review]
Thank you, this looks great but it seems there is a leftover printf :)

File: boot/assets_test.go (hunk 1 above)
Comment (mvo5) [reply boot/assets_test.go]: I removed it (as it's removed in https://github.com/snapcore/snapd/pull/12769 too)

**Review by pedronis — APPROVED** [review]
thanks

**Pull request merged**